week-3/Homework/mandatory/2-bush-berries.js

```diff
@@ -10,8 +10,10 @@
   Use the tests to confirm which message to return
 */
 
-function bushChecker() {
+console.clear(); // Clear all old outputs and see your new output.
 
+function bushChecker(arry) { // This function is for check bushs
+  return (arry.filter(Element => Element != "pink")).length > 0 ? "Toxic! Leave bush alone!" : "Bush is safe to eat from"; // This line is for check berries is safe or not safe
```

Comment:
Using the word `Element` is potentially troublesome as there is a [keyword in browsers called that](https://developer.mozilla.org/en-US/docs/Web/API/Element).

It is common to try keep your casing of variables the same as the code. i.e. `element`, or to even be more perspective and call is berry.